webserver/webserver/src/main/java/io/helidon/webserver/WebServerTls.java

```diff
@@ -95,12 +96,62 @@ public static WebServerTls create(Config config) {
         return builder().config(config).build();
     }
 
+    /**
+     * The Tls manager. If one is not explicitly defined in the config then a default manager will be created.
+     *
+     * @return the tls manager of the tls instance
+     * @see ConfiguredTlsManager
+     */
+    public TlsManager manager() {
+        return tlsManager;
+    }
+
+    /**
+     * Trust any certificate provided by the other side of communication.
+     * <p>
+     * <b>This is a dangerous setting: </b> if set to {@code true}, any certificate will be accepted, throwing away
+     * most of the security advantages of TLS. <b>NEVER</b> do this in production.
+     *
+     * @return whether to trust all certificates, do not use in production
+     */
+    public boolean trustAll() {
+        return trustAll;
+    }
+
     Collection<String> enabledTlsProtocols() {
         return enabledTlsProtocols;
     }
 
+    Optional<SSLContext> explicitSslContext() {
+        return Optional.ofNullable(explicitSslContext);
+    }
+
     SSLContext sslContext() {
-        return sslContext;
+        if (explicitSslContext != null) {
+            return explicitSslContext;
+        }
```

Comment:
If someone explicitly sets the `SSLContext` programmatically, we want to take it as-is. The `ConfiguredTlsManager` should not be redefined from config as that would give a higher weight to config over the programmatic approach. What I can do is log a warning when programmatic is set and the `OciTlsManager` is being used - because that is nonsensical.